src/tree/Treemap.css

```diff
@@ -0,0 +1,8 @@
+﻿.tree_Treemap .node {
+    border: solid 1px white;
+    font: 10px sans-serif;
```

Comment:
This will override fonts set by the including site do you want to do that?  Also it would be nice if fonts were exposed as a published param.